src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -987,10 +991,45 @@ void SlurTieLayout::fixArticulations(Slur* item, PointF& pt, Chord* c, double up
     }
 }
 
-//---------------------------------------------------------
-//   layoutFor
-//    layout the first SpannerSegment of a slur
-//---------------------------------------------------------
+TieSegment* SlurTieLayout::layoutTieWithNoEndNote(Tie* item)
+{
+    StaffType* st = item->staff()->staffType(item->startNote()->tick());
+    Chord* c1 = item->startNote()->chord();
+    item->setTick(c1->tick());
+
+    if (item->slurDirection() == DirectionV::AUTO) {
+        bool simpleException = st && st->isSimpleTabStaff();
+        if (simpleException) {
+            item->setUp(isUpVoice(c1->voice()));
+        } else {
+            if (c1->measure()->hasVoices(c1->staffIdx(), c1->tick(), c1->actualTicks())) {
+                // in polyphonic passage, ties go on the stem side
+                item->setUp(c1->up());
+            } else {
+                item->setUp(!c1->up());
+            }
+        }
+    } else {
+        item->setUp(item->slurDirection() == DirectionV::UP ? true : false);
+    }
+
+    item->fixupSegments(1);
```

Comment:
What does 1 means?

Reply:
It means that the Tie only needs 1 TieSegment, which is correct in this case